common/defs/bitcoin/gleacebtc.json

```diff
@@ -0,0 +1,47 @@
+{
+  "coin_name": "GleecBTC",
+  "coin_shortcut": "Gleec",
+  "coin_label": "GleecBTC Coin",
+  "website": "https://gleecoin.com",
+  "github": "https://github.com/Gleecs/GleecBTC-FullNode-Win-Mac-Linux",
+  "maintainer": "Jochen Hoenicke <hoenicke@gmail.com>",
```

Comment:
This is wrong - put your own email here.

Reply:
Update Done. Also, mistake was File name